app/experimenter/experiments/tests/test_models.py

```diff
@@ -1115,6 +1115,106 @@ def test_monitoring_dashboard_url_returns_url_when_experiment_is_complete(self):
             ),
         )
 
+    def test_rollouts_monitoring_dashboard_url_is_none_when_rollout_not_begun(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            slug="rollout-1-slug",
+            status=NimbusExperiment.Status.DRAFT,
+        )
+        self.assertIsNone(experiment.rollout_monitoring_dashboard_url)
+
+    def test_rollouts_monitoring_dashboard_url_is_none_on_first_day(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            slug="rollout-1-slug",
+            status=NimbusExperiment.Status.DRAFT,
+        )
+
+        NimbusChangeLogFactory.create(
+            experiment=experiment,
+            old_status=NimbusExperiment.Status.DRAFT,
+            new_status=NimbusExperiment.Status.LIVE,
+            changed_on=datetime.date.today(),
+        )
```

Comment:
Oh yeah, this isn't really the right way to do this.  Then you've got an experiment whose status is `DRAFT` but has a `DRAFT` to `LIVE` changelog, so the experiments in kind of a wonky state, which could produce other weird unintended behaviours.  What you'd wanna do is use the lifecycle that puts it into the end state you want, and then lookup the changelog for `DRAFT` to `LIVE` and update its date.

Reply:
ohhh I see what you mean 👍